test/spec/BpmnPropertiesPanelRenderer.spec.js

```diff
@@ -518,6 +518,41 @@ describe('<BpmnPropertiesPanelRenderer>', function() {
       expect(getHeaderName(propertiesContainer)).to.eql('start');
     });
 
+
+    it('should show error', async function() {
```

Comment:
I was a bit too quick, this one fails for properties panel versions < 16 (cf. https://github.com/bpmn-io/bpmn-js-properties-panel/runs/7252180335?check_suite_focus=true#step:7:64).

We can use [our helpers](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/test/TestHelper.js#L169) for this

```suggestion
     withPropertiesPanel('>=0.16')('should show error', async function() {
```